pkg/config/transport.go

```diff
@@ -124,5 +174,35 @@ func serverFinalizer(logger log.Logger) kithttp.ServerFinalizerFunc {
 				"statusCode": code,
 			},
 		)
+
+		requestLatency := kitprom.NewHistogramFrom(
+			prometheus.HistogramOpts{
+				Namespace: namespace,
+				Subsystem: subsystemRequest,
+				Name:      "req_latency_seconds",
+				Help:      "Total duration of requests in seconds",
+			},
+			requestLabels,
+		)
+
+		requestLatency.With(
+			labelComponent, "request",
+			labelRoute, route,
+			labelStatus, statusCode,
```

Comment:
It would be helpful to add labels with low cardinality like: `host`, `method` and `proto`.